---
layout: news
title: "IBSE Seminar Series: <em>What transcriptional noise tells us about the cell inside</em>"
author: "Karthik Raman"
author_handle: kraman
image: /assets/images/news/victor.jpg
category: news
tags: [seminars]
brief: The first talk of the IBSE seminar series will be delivered by Prof. Victor de Lorenzo.
---
## What transcriptional noise tells us about the cell inside

**Speaker: Victor de Lorenzo**

**Venue: Central Lecture Theatre (CLT), at 3:30 pm on Friday, 3rd November 2017**

_Systems Biology Program. Centro Nacional de Biotecnologia, CSIC, C/ Darwin, 3
(Campus de Cantoblanco), Madrid 28049, Spain._ 

### Abstract

Transcriptional noise is a necessary consequence of the molecular events that drive gene
expression in prokaryotes. However, the _m_-xylene degrading soil bacterium _Pseudomonas
putida_ mt-2 seem to have co-opted evolutionarily such a noise for deploying a metabolic
diversification strategy that allows a cautious exploration of new chemical landscapes. In
reality, noise is not only the mere consequence of stochasticity, but also a signal that
reflects the upstream physical dynamics of the cognate molecular machinery. Noise of
catabolic promoters can thus be exploited to deploy beneficial phenotypes such as
metabolic bet-hedging and/or division of biochemical labour. Although the role of upstream
promoter-regulator interplay in the origin of this noise is little understood, its specifications
are probably ciphered in flow cytometry data patterns. We studied the activity of the _Pm_
promoter of _Pseudomonas putida_ mt-2 and its cognate regulator XylS by following
expression of _Pm-gfp_ fusions in single cells. Using mathematical modelling and
computational simulations, we determined the kinetic properties of the system and used
them as a baseline code to interpret promoter activity in terms of upstream regulator
variability. Transcriptional noise was predicted to depend on the intracellular physical
distance between regulator source (where XylS is produced) and the target promoter.
Experiments with engineered bacteria in which this distance is minimised or enlarged
confirmed the predicted effects of source/target proximity on noise patterns. This approach
allowed deconvolution of cytometry data into mechanistic information on gene expression
flow. It also provided a basis for selecting programmable noise levels in synthetic
regulatory circuits.



**Goñi-Moreno Á, Benedetti I, Kim J, de Lorenzo V.** (2017) Deconvolution of Gene Expression Noise into
Spatial Dynamics of Transcription Factor-Promoter Interplay. _ACS Synth Biol._ 6: 1359-1369

**Guantes R, Benedetti I, Silva-Rocha R, de Lorenzo V.** (2016) Transcription factor levels enable metabolic
diversification of single cells of environmental bacteria. _ISME J._ 10: 1122-1133

**Silva-Rocha, R. and de Lorenzo, V.** (2012) Stochasticity of TOL plasmid catabolic promoters sets a
bimodal expression regime in _Pseudomonas putida_ mt-2 exposed to _m_-xylene. _Mol. Microbiol._ 86: 199-
211



